README.md

```diff
@@ -24,6 +24,55 @@ Convenient and minimalistic DNS based redirects
 
 Take a look at our full [documentation](/docs).
 
+## Examples
+
+Redirect requests from example.com to www.example.com:
+
+```
+txtdirect {
+    enable www
+}
+```
+
+Redirect to host provided in TXT record if one is found, otherwise 404:
+
+```
+txtdirect {
+    enable host
+}
+```
+
+Redirect to host provided in TXT record if one is found, otherwise redirect to www.myhost.com:
```

Comment:
I think we should stick with www.example.com. I see why you want to use myhost.com, but there might be a better way to document this.

Reply:
Right! I wasn't sure about which one to use.